src/components/app-configure/access/DidDelegate.vue

```diff
@@ -20,14 +24,72 @@ const loaderStore = useLoaderStore()
 const toast = useToast()
 
 const app = appsStore.app(appId)
-const delegates = computed(() => app.access.delegates)
+const delegates = ref(app.access.delegates)
 
-const showDeletePopup = ref(false)
+type selectedOperation = 'create' | 'edit'
```

Comment:
I think we might have this type defined somewhere else too. 

```suggestion
type DelegateOperation = 'create' | 'edit'
```